core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/EntityStatisticsCacheMonitor.scala

```diff
@@ -28,6 +28,8 @@ object EntityStatisticsCacheMonitor {
 
   //Note: Ignored workspaces have a cacheLastUpdated timestamp of 1000ms after epoch
   val MIN_CACHE_TIME = new Timestamp(1000)
+  // when resetting an invalid cache so that it can be recalculated, use this timestamp
+  val RESET_CACHE_TIME = new Timestamp(2000)
```

Comment:
I'm not a fan of magic values.  I understand that doing it this way means the background cache refresh operation code doesn't have to change, but I think we should be more explicit and use columns that specifically say the cache needs refreshing and in what order.

Reply:
I agree - but as before, I'm hoping that work happens in AS-931. I realize this is adding even more tech debt for AS-931 to fix, but I wanted this PR to focus on the admin APIs and not on the cache schema.